frontend/src/components/MultiSelect.js

```diff
@@ -105,8 +106,8 @@ function MultiSelect({
     }
     return value.map((item) => ({
       ...item,
-      label: item[labelProperty],
-      value: item[valueProperty],
+      label: _.get(item, labelProperty),
+      value: _.get(item, valueProperty),
```

Comment:
Curious - any reason lodash was needed here?

Reply:
This was changed to use loddash in order to access properties on the User object of the approver (approver.User.id).